packages/features/handleCreatePhoneCall.ts

```diff
@@ -0,0 +1,128 @@
+import { PROMPT_TEMPLATES } from "@calcom/features/ee/cal-ai-phone/promptTemplates";
+import { RetellAIService, validatePhoneNumber } from "@calcom/features/ee/cal-ai-phone/retellAIService";
+import { templateTypeEnum } from "@calcom/features/ee/cal-ai-phone/zod-utils";
+import type { TCreatePhoneCallSchema } from "@calcom/features/ee/cal-ai-phone/zod-utils";
+import { checkRateLimitAndThrowError } from "@calcom/lib/checkRateLimitAndThrowError";
+import logger from "@calcom/lib/logger";
+import prisma from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+export const handleCreatePhoneCall = async ({
+  user,
+  input,
+}: {
+  user: { timeZone: string; id: number; profile?: { organization?: { id?: number } } };
+  input: TCreatePhoneCallSchema;
+}) => {
+  if (!!!user?.profile?.organization) {
+    throw new TRPCError({ code: "UNAUTHORIZED" });
+  }
```

Comment:
is user should be part of an org to use this endpoint, we should create a guard for it